examples/spire-plugins/node-agent-tpm/values.yaml

```diff
@@ -23,3 +23,15 @@ spire-server:
         plugin_data:
         - ca_path: /run/spire/data/certs
           hash_path: /run/spire/data/hashes
+
+spire-agent:
+  image:
+    registry: docker.io
+    repository: kfox1111/misc
```

Comment:
Should we use an official image here?

Reply:
There isn't one. Its the official image + the tpm plugin like in the Dockerfile. We could remove it and just put a placeholder there too. <your agent image here> ?

Reply:
removed it in favor of something the user has to specify